css/style.less

```diff
@@ -280,6 +280,16 @@ form.lms-ui-quick-search {
 	margin-left: auto;
 }
 
+.lms-ui-relatedticket {
+	color: darkgreen !important;
+	text-decoration: none;
+}
+
+.lms-ui-parentticket {
+	color: darkred !important;
+	text-decoration: none;
+}
+
 #lms-ui-welcome-boxes {
```

Comment:
Efekt tego będzie taki, że potem jak będziesz chciał zmienić w jednym miejscu kolory dla całej instalacji będzie trzeba przeglądać cały plik less, zamiast tylko lms-variables.less. Zastanów się jak to zrobić porządnie.

Reply:
Wyobraź sobie, że w lms-variables.less zmieniamy kolory na odcienie zielonego dla całego LMS, a tu taki kwiatek - kolor, którego nie można odróżnić od reszty.

Reply:
A !important pewnie dane dlatego, że nie chciało bez tego działać i dało się na wszelki wypadek? Efekt tego będzie taki, że za jakiś czas wymuszanie styli będzie tak uciążliwe, że będzie trudno cokolwiek zmienić...

Reply:
Z Rafałem uważamy, że kolory ikon powinny być, albo czarne, albo białe - zależnie od tematu graficznego. To co obecnie jest to jest pstrokacizna. Wiele osób tak uważa również.

Reply:
Przy kolorowych ikonach ciężko też stany przycisków linków jednoznacznie określać. Masz guzik z ikonką zablokowany i jak kolor to ma wskazywać? A tak masz czarny - normalny stan, szary - zablokowany stan. Zdecydowanie te kolory różne źle wyglądają.

Reply:
Przespałem utworzenie tego pliku:

czyli w less/lms-variables:
`@darkred #8B0000`

w css/style.less:
zmiana z `darkred` na `@darkred`

tak?

Reply:
OOoo wyświetliło mi sie reszte komentarzy dopiero

Reply:
Przeglądałem kilku produktów konkurencji - nikt oczojebnych kolorowych ikon nie używa. Jak myślisz - dlaczego?

Reply:
Na demo.lms.org.pl wyłączyłem tymczasowo kolorowanie ikon. Spytam parę osób co myślą o takiej zmianie.

Reply:
> Przeglądałem kilku produktów konkurencji - nikt oczojebnych kolorowych ikon nie używa. Jak myślisz - dlaczego?

Produkty konkurencji mają duże ikonki więc color coding nie jest aż tak potrzebny bo kształt-coding jest wystarczający.

Moim wzorem jest iPKO i ING. Uważam, że te interfejsy są na prawdę super zaprojektowane. A Ty o jakich mówisz?

Reply:
> Z Rafałem uważamy, że kolory ikon powinny być, albo czarne, albo białe - zależnie od tematu graficznego. To co obecnie jest to jest pstrokacizna. Wiele osób tak uważa również. No i przez pstrokaciznę utrzymanie staje się mocno złożone.

Uważam, że sam color-coding jest jak najbardziej potrzebny bo mózg ludzki szybciej interpretuje kolory niż tak złożone ikonki (kształt-koding). Tylko ikonki nie są jedynym elementem, który może skupiać uwagę na poszukiwanym elemencie. Może to być też nagłówek tabeli np w customerinfo możnaby pokolorować np. lms-box-titlebar i lms-ui-tab-row-header ... gdyby te bary pokolorować to faktycznie ikonki tam mogą nie mieć kolorów.

Jeśli chodzi o ikonki w rtticketview w tabeli z informacjami o tickecie zdjąłbym z linków kolory na początek i dekoracje bo tam są zdbędne a na ikonkach bym je jednak zostawił bo tekstu jest dużo. Tak samo jak na informacjach o najważniejszych właściwościach ticketu np stan ticketu (nowy - bordo, oczekujące - niebieskie) głównie, żeby łatwo odróżnić to co jest opisane w definitions.php.

Pstrokacizna trochę jest ale w innych elementach UI nie w samych ikonkach, które po wymianie na fontawesome są bardziej spójne.

> Przy kolorowych ikonach ciężko też stany przycisków linków jednoznacznie określać.

Nie zgodzę się.

Stany kontrolek powinny odznaczać się:
- kolorem,
- przeźroczystością/szarością,
- znacząco inny kształt (inna ikonka),
- wypełnienie?

Najlepiej, żeby były to te wszystkie cechy na raz. 

Natomiast ikonki bez stanowe w stylu: Edytuj, Wydrukuj, Usuń, Nowa Notatka, Dodaj, Wiadomość (np na liście ticketów) - faktycznie mogą mieć jeden kolor i byłoby to wskazane w obrębie jednego panelu przycisków ale gdy będą 3x większe. Bo w tym momencie przy dużej rozdzielczości nie można ich odróżnić inaczej niż po kolorach i po najechaniu myszką.

Reply:
Proponuję w takim razie, żebyś zaczął utrzymywać własnego forka.

Reply:
Generalnie sprawa jest prosta - trzeba zapytać parę osób co myślą o wyglądzie pstrokatym, a co o normalnym. Nie podawaj mi proszę jako wzorców iPKO, bo to akurat państwowy szajs, który nie rządzi się prawami rynkowymi. Wzorem i zwycięzcą wielu konkursów jest mBank. Zobacz jakie kolory ikon ma github...

Reply:
> Przespałem utworzenie tego pliku:
> 
> czyli w less/lms-variables.less:
> `@darkred #8B0000`
> 
> w css/style.less:
> zmiana z `darkred` na `@darkred`
> 
> tak?

Bzdura - co to zmieni - nazwy zmiennych powinny być semantyczne, a nie uzależnione od koloru. Co to zmieni jak wrzucić definicję 150 kolorów ikon do lms-variables.less? Żeby zmienić ogólny temat graficzny będziesz musiał rozgryzać zmianę 150+30 zmiennych. Bez sensu.

Reply:
> Generalnie sprawa jest prosta - trzeba zapytać parę osób co myślą o wyglądzie pstrokatym, a co o normalnym. 

Może lepiej zadaj pytanie co myślą o wyglądzie grobowym a co o normalnym. Ostatnia zmiana czyli brak kolorów w menu głównym sprawia, że wyszukiwanie poszukiwanej opcji jest wolniejsze.

> Nie podawaj mi proszę jako wzorców iPKO, bo to akurat państwowy szajs, który nie rządzi się prawami rynkowymi. 

Proszę nie wplataj polityki w dyskusję - ten interfejs używa się najlepiej z tych co próbowałem (~6 banków).

> Wzorem i zwycięzcą wielu konkursów jest mBank. 

Możliwe. Jakieś 7 lat temu miałem mBank - ale wtedy był denny do tego stopnia że każda czynność zajmowała mi 2 razy dłużej niż gdzieś indziej. Nie wiem jak teraz bo nie mam ale stronę WWW mają ładną.

> Zobacz jakie kolory ikon ma github...

@chillek proponuję, żebyś naszkicował jakiś przykładową makietę porządanego stanu intefejsu bo takie rzeczy należy omawiać wraz z szerszym kontekstem. Bez tego dyskusja nad tym czy tęczowe/grobowe ikonki są passe jest jałowa.

Reply:
Sorry nie kupuję takiej argumentacji. Firefox też jest lepszy od Google Chrome (może w jakimś zakresie tak, ale w zakresie wydajność Firefox to muł: https://www.phoronix.com/scan.php?page=article&item=chrome-75-linux&num=1). Poczekajmy co z ankiety wyniknie. Większość specjalizowanych aplikacji nie bawi się w kolorowanie ikon.

Reply:
Co do przeglądarki - używam obu zamiennie. Nie widzę, żeby któraś była znacząco szybsza - może to kwestia tego, że jednak mam dość mocny sprzęt. 

Stawiasz na szali te: kilkadziesiąt ms kosztem jakiejś dużej części sekundy przy wyszukiwaniu okiem interesującej opcji usuwając color-coding. Wydajność interfejsu lepiej badać nie na poziomie samej aplikacji ale całej czynności od rozpoczęcia do zakończenia.

Żebyś mnie źle nie zrozumiał jeszcze raz to napiszę - nie mówię, że ikonki kolorowe/czarne są złe czy dobre bo to cecha a nie wada/zaleta więc: 
@chilek proponuję, żebyś naszkicował jakiś przykładową makietę pożądanego stanu intefejsu bo takie rzeczy należy omawiać wraz z szerszym kontekstem. Bez tego dyskusja nad tym czy tęczowe/grobowe ikonki są passe jest jałowa.

Gdzie ta ankieta?